src/doc/rustc/src/platform-support/unknown-uefi.md

```diff
@@ -196,8 +195,8 @@ pub extern "C" fn main(_h: *mut core::ffi::c_void, _st: *mut core::ffi::c_void)
 }
 ```
 
-## Example: Hello World
 
+## Example: Hello World
```

Comment:
Looks like a spurious empty-line change. Now you have a double-empty-line above the header?